lib/services/common/securityServiceOAuth2.js

```diff
@@ -64,17 +71,27 @@ function auth(trust, callback) {
     };
 
     logger.debug(context, 'Authentication on the OAuth2 provider [%s] using trust [%s]', url, trust);
+    logger.debug(context, 'Using the following request:\n\n%s\n\n', JSON.stringify(options, null, 4));
```

Comment:
The client secret is being pushed to the debug log in plain text. This is a security leak.

Reply:
LGTM